app/views/admin/OutcomesReportComponent.vue

```diff
@@ -4,8 +4,8 @@
   form
     label
       | Teacher email:
-      input(v-model.trim="teacherEmail", :disabled="teacher")
-    button(type="submit" @click.prevent="submitEmail")
+      input(v-model.trim="teacherEmail", :disabled="dataFetching")
```

Comment:
Why not `!(dataReady)`?

Reply:
It would be disabled when the page loads because dataReady=false